src/foam/nanos/crunch/RenewableData.js

```diff
@@ -37,6 +37,7 @@ foam.CLASS({
       class: 'Boolean',
       label: '',
       section: 'reviewDataSection',
+      value: true,
```

Comment:
wouldn't this mean that it would be visible also when not in a renewable state?

Reply:
No, the visibility of `renewDataSection` is based on the value of `renewable` not `reviewed`.